k8s/aat/common/xui/xui-tc.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: xui-terms-and-conditions
+  namespace: xui
+  annotations:
+    fluxcd.io/automated: "true"
+    filter.fluxcd.io/tag.node: glob:prod-*
+spec:
+  releaseName: xui-terms-and-conditions
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: terms-and-conditions
+    version: 1.0.0
```

Comment:
This should match https://github.com/hmcts/rpx-xui-terms-and-conditions/blob/ithc/charts/xui-terms-and-conditions/Chart.yaml#L4

Reply:
updated version as requested